tests/serialized_test_data_generation/Makefile

```diff
@@ -1,69 +1,123 @@
-	if [ ! -d $(RUNDIR_HOST) ]; then \
-	write_run_directory configs/$(EXPERIMENT).yml $(RUNDIR_HOST) && \
-	cp submit_job.sh $(RUNDIR_HOST) && cp count_ranks.py $(RUNDIR_HOST) && \
-	git rev-parse HEAD > $(RUNDIR_HOST)/fortran_sha.txt ; fi
+SHELL = /bin/bash
 
-compile_fortran:
-	cd $(FORTRAN_DIR) && DOCKER_BUILDKIT=1 SERIALIZE_IMAGE=$(COMPILED_IMAGE) $(MAKE) build_serialize_gt4py_dev
+# select configuration file (has to be in ./configs)
+CONFIGURATION ?= c12_6ranks_standard
 
-run_model_make_data:
-	docker run --network host --rm -v $(RUNDIR_HOST):$(RUNDIR_CONTAINER) -v $(TEST_DATA_HOST):$(TEST_DATA_CONTAINER) -it $(COMPILED_IMAGE)-gt4pydev /rundir/submit_job.sh
-
-generate_test_data: setup_rundir compile_fortran run_model_make_data
-
-pack_test_data:
-	if [ ! -d $(TEST_DATA_HOST)/$(TEST_DATA_TARFILE) ]; then \
-	cd $(TEST_DATA_HOST) && tar  --exclude=input.nml --exclude=*.json  --exclude=$(TEST_DATA_TARFILE) \
-	-czf $(TEST_DATA_TARFILE) . ;fi 
-
-	for number in 0 1 2 3 4 5 6 7 8 9 ; do \
-		cd $(TEST_DATA_HOST) && rm -f Gen*rank$$number*.dat ; \
-	done
-unpack_test_data:
-	if [ -f $(TEST_DATA_TARPATH) ]; then \
-	cd $(TEST_DATA_HOST) && tar -xf $(TEST_DATA_TARFILE) && \
-	rm $(TEST_DATA_TARFILE); fi
-
-push_test_data:
-	echo $(LOOKUP_BUCKET)
-	gsutil -m rsync $(TEST_DATA_HOST) $(TEST_DATA_BUCKET)	
-
-generate_and_push_data: setup_rundir run_model_make_data pack_test_data push_test_data
-
-generate_coverage: install_deps
-	rm -rf coverage
-	EXPERIMENT=fv3config_coverage $(MAKE) setup_rundir 	
-	cd $(FORTRAN_DIR) && DOCKER_BUILDKIT=1 $(MAKE) build_coverage
-	mkdir coverage
-	docker run -it --rm --network host -v $(PWD)/coverage:/coverage -v $(CWD)/rundir/fv3config_coverage:$(RUNDIR_CONTAINER) $(GCR_URL)/fv3gfs-compiled:gcov bash -c "pip install gcovr; cd /coverage; mkdir physics; cd physics; gcovr -d -r /FV3/gfsphysics --html --html-details -o index.html; cd ../; mkdir dycore; cd dycore; gcovr -d -r /FV3/atmos_cubed_sphere --html --html-details -o index.html"
-	@echo "==== Coverage ananlysis done. Now open coverage/dycore/index.html coverage/physics/index.html in your browser ===="
+# version number or tag (determines image tag and destination directory in cloud storage)
+# -convention: <tag>-<some large conceptual version change>.<serialization statement change>.<hotfix>
+# -version number should be increased when serialization data is expected to change
+# -omit tag for "operational" serialization data, use tag for experimental serialization data
+FORTRAN_VERSION ?= jenkins-7.1.1
 
+# docker container image setup (used for running model, see ../../README.md)
+GCR_URL = us.gcr.io/vcm-ml
+COMPILED_IMAGE = $(GCR_URL)/fv3gfs-compiled:$(FORTRAN_VERSION)-serialize-gt4pydev
+RUN_DIR_CONTAINER = /rundir
+DATA_DIR_CONTAINER = $(RUN_DIR_CONTAINER)/test_data
 
+# local host setup
+CWD=$(shell pwd)
+FORTRAN_DIR = $(CWD)/../../
+CONFIGURATION_DIR = $(CWD)/configs
+RUN_DIR_HOST = $(CWD)/rundir/$(CONFIGURATION)
+DATA_DIR_HOST = $(CWD)/data/$(CONFIGURATION)
```

Comment:
It made naming shorter and - imho - did not decrease information content. It also allowed me to make some other variable namings consistent without length of variable names going overboard. If you agree, I would propose that it's fine to call it data here and test_data in fv3core, since here it's not really related to 'testing'.